examples/partitioned-heat-conduction/README.md

```diff
@@ -0,0 +1,68 @@
+
+## Setup
+
+We solve a partitioned heat equation. For information on the non-partitioned case, please refer to [1, p.37ff]. In this tutorial the computational domain is partitioned and coupled via preCICE. The coupling roughly follows the approach described in [2].
+
+![Case setup of partitioned-heat-conduction case](https://github.com/precice/tutorials/blob/master/partitioned-heat-conduction/images/tutorials-partitioned-heat-conduction-setup.png)
+
+Case setup from [2]. `D` denotes the Dirichlet participant and `N` denotes the Neumann participant.
+
+The heat equation is solved on a rectangular domain `Omega = [0,2] x [0,1]` with given Dirichlet boundary conditions. We split the domain at `x_c = 1` using a straight vertical line, the coupling interface. The left part of the domain will be referred to as the Dirichlet partition and the right part as the Neumann partition. To couple the two participants we use Dirichlet-Neumann coupling. Here, the Dirichlet participant receives Dirichlet boundary conditions (`Temperature`) at the coupling interface and solves the heat equation using these boundary conditions on the left part of the domain. Then the Dirichlet participant computes the resulting heat flux (`Flux`) from the solution and sends it to the Neumann participant. The Neumann participant uses the flux as a Neumann boundary condition to solve the heat equation on the right part of the domain. We then extract the temperature from the solution and send it back to the Dirichlet participant. This establishes the coupling between the two participants.
+
+This simple case allows us to compare the solution for the partitioned case to a known analytical solution (method of manufactures solutions, see [1, p.37ff]). For more usage examples and details, please refer to [3, sect. 4.1].
+
+## Configuration
+
+preCICE configuration (image generated using the [precice-config-visualizer](https://precice.org/tooling-config-visualization.html)):
+
+![preCICE configuration visualization](https://github.com/precice/tutorials/blob/master/partitioned-heat-conduction/images/tutorials-partitioned-heat-conduction-precice-config.png)
+
+
+## Running the simulation
+
+In the G+Smo build folder, build the tutorial file.
+
+```
+make partitioned-heat-conduction -j <#threads>
+make partitioned-heat-conduction-IGA-dirichlet -j <#threads>
+make partitioned-heat-conduction-IGA-neumann -j <#threads>
+```
+
+Go to the gismo-executable folder and link the compiled executable to the gismo_executable.
```

Comment:
is this needed? 
What happens if you call `make install`  in your build folder?

Reply:
Solved.Use
```
make <participant file> -j <#threads>
```